requirements-test.txt

```diff
@@ -1,32 +1,40 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.6
```

Comment:
Do we really want to do it with python3.6?

Reply:
Yes, ma'am. IIB runs on python 3.6 in our deployments. We have also added the [.python-version](https://github.com/release-engineering/iib/blob/master/.python-version) file so dependabot uses python 3.6 as well.

A new version of pip-tools must be writing the python version now. Thus the difference.

Reply:
ok, thanks